zone/gm_commands/petname.cpp

```diff
@@ -2,21 +2,44 @@
 
 void command_petname(Client *c, const Seperator *sep)
 {
-	Mob *target;
-	target = c->GetTarget();
+	Mob *t = nullptr;
+	if (c->GetTarget()) {
+		t = c->GetTarget();
+	}
 
-	if (!target || !target->IsPet()) {
-		c->Message(Chat::White, "Usage: #petname newname (requires a pet target)");
```

Comment:
We should include a help message when no argument provided. Also mention that the name change is temporary because its not permanent